lib/bambora/client/version.rb

```diff
@@ -2,6 +2,6 @@
 
 module Bambora
   class Client
-    VERSION = '0.1.3'
+    VERSION = '0.2.0'
```

Comment:
Why does this get updated?

Reply:
I updated this manually. I am increasing the gem version number to signify new changes are present with this PR. This will also deploy the new version to RubyGems with a GitHub action.